### PR TITLE
Use our eslint rules with storybook [SATURN-1201]

### PR DESCRIPTION
**Pull request by petesantos**
Correct issue with storybook build around eslint problems. This change applies our eslint rules to our storybook code

### CODE REVIEW

**Review by panentheos — APPROVED** [review]
Thanks for tracking this down!

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 const path = require('path')
+const { useEslintRc } = require('customize-cra')
```
Comment (zarsky-broad) [inline .storybook/webpack.config.js:2]:
```suggestion
const { useEslintRc } = require('customize-cra')

```

File: .storybook/webpack.config.js [hunk 2]
```diff
@@ -15,5 +14,7 @@ module.exports = ({ config }) => {
     use: [{ loader: require.resolve('svg-react-loader') }]
   })
 
-  return config
+  const newConfig = useEslintRc()(config)
+
+  return newConfig
```
Comment (zarsky-broad) [inline .storybook/webpack.config.js]: why not just `return useEslintRc()(config)`?

File: .storybook/webpack.config.js [hunk 3]
```diff
@@ -1,10 +1,9 @@
 const path = require('path')
+const { useEslintRc } = require('customize-cra')
 
 module.exports = ({ config }) => {
   // Needed for .stories/config.js
   config.resolve.modules.push(path.resolve(__dirname, '../'))
-
-  // Replace storybook's default svg loader
```
Comment (zarsky-broad) [inline .storybook/webpack.config.js:7]: Why delete this comment?

**Review by zarsky-broad — APPROVED** [review]

**Pull request merged**